taccsite_cms/contrib/helpers.py

```diff
@@ -0,0 +1,68 @@
+# Get Django `models.CharField` `choices`
+def get_choices(choice_dict):
+    """Get a sequence for a Django model field choices from a dictionary.
+    :param Dict[str, Dict[str, str]] dictionary: choice as key for dictionary of classnames and descriptions
+    :return: a sequence for django.db.models.CharField.choices
+    :rtype: List[Tuple[str, str], ...]
+    """
+    choices = []
+
+    for key, data in choice_dict.items():
+        choice = (key, data['description'])
+        choices.append(choice)
+
+    return choices
+
+
+
+# GH-93, GH-142, GH-133: Upcoming functions here (ease merge conflict, maybe)
```

Comment:
__Note__: This `helpers.py` file has functions from PR #280 that this PR's code also needs. So, I copied them over.